streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionVertex.java

```diff
@@ -37,17 +38,33 @@
    */
   private final Map<String, Double> resources;
 
+  private final StreamingWorkerConfig workerConfig;
+
+  private final StreamOperator streamOperator;
+
   private ExecutionVertexState state = ExecutionVertexState.TO_ADD;
   private Slot slot;
   private RayActor<JobWorker> workerActor;
   private List<ExecutionEdge> inputEdges = new ArrayList<>();
   private List<ExecutionEdge> outputEdges = new ArrayList<>();
-
-  public ExecutionVertex(int jobVertexId, int index, ExecutionJobVertex executionJobVertex) {
+  private int parallelism;
+  private long buildTime;
```

Comment:
Why not set buildTime in global config?
It's redundant to hold the same field in every executionvertex.